misk/web/@misk/common/src/vendors.js

```diff
@@ -1,5 +1,5 @@
-window.BlueprintCore = require('@blueprintjs/core')
-window.BlueprintIcons = require('@blueprintjs/icons')
+window.BlueprintCore = require('@blueprintjs/core');
+window.BlueprintIcons = require('@blueprintjs/icons');
```

Comment:
Linter didn't catch this earlier?

Reply:
Unfortunately not. Linting currently only runs on Typescript files since every file in the repo except this one is Typescript.